docker-compose.yaml

```diff
@@ -44,6 +44,7 @@ x-app: &common
       - WORKSPACES_DATABASE_HOST=db
       - WORKSPACES_DATABASE_PORT=5432
       - WORKSPACES_DATABASE_DEFAULT_DB=hexa-app
+      - WORKSPACES_DATABASE_PROXY_URL=explore.openhexa.org
```

Comment:
The proxy does not exist in dev but we could try with `db` (the name of the service in the docker-compose.yaml)